runtimes/eden/src/migrations.rs

```diff
@@ -0,0 +1,57 @@
+use crate::Runtime;
+use frame_support::{migration, traits::OnRuntimeUpgrade, weights::Weight};
+
+#[cfg(feature = "try-runtime")]
+use {
+	codec::{Decode, Encode},
+	frame_support::Blake2_128Concat,
+	pallet_uniques::CollectionDetails,
+	primitives::{AccountId, Balance},
+	sp_std::prelude::*,
+};
+
+#[cfg(feature = "try-runtime")]
+type CollectionId = u32;
+
+#[cfg(feature = "try-runtime")]
+const UNIQUES_CLASS_PREFIX: &[u8] = b"Class";
+
+const NEW_UNIQUES_PALLET_NAME: &[u8] = b"SubstrateUniques";
+const OLD_UNIQUES_PALLET_NAME: &[u8] = b"Uniques";
+
+pub struct MovePalletUniquesToSubstrateUniques;
+impl OnRuntimeUpgrade for MovePalletUniquesToSubstrateUniques {
+	fn on_runtime_upgrade() -> Weight {
```

Comment:
How does this trigger...

- [ ] check trigger
- [ ] run and upgrade on local testnet